extra/man/alacritty.5.scd

```diff
@@ -144,19 +148,26 @@ This section documents the *[window]* table of the configuration file.
 	Override the variant of the System theme/GTK theme/Wayland client side
 	decorations. Set this to _"None"_ to use the system's default theme variant.
 
-	Default: _"None"_
-
 *resize_increments* <boolean>
 
 	Prefer resizing window by discrete steps equal to cell dimensions.
 
-	Default: _false_
-
 *option_as_alt* "OnlyLeft" | "OnlyRight" | "Both" | "None" _(macos only)_
 
 	Make _Option_ key behave as _Alt_
 
-	Default: _"None"_
+Default:
+
+	*[window]*++
+dimensions      = { columns = _0_, lines = _0_ }++
+padding         = { x = _0_, y = _0_ }++
+dynamic_padding = _false_++
+decorations     = _"Full"_++
+opacity         = _1.0_++
+startup_mode    = _"Windowed"_++
+title           = _"Alacritty"_++
+dynamic_title   = _true_++
+decorations_theme_variant = _"None"_
```

Comment:
Not a fan of having all the defaults documented in a massive blog so far from its definition.

I don't want to scroll through half a manpage just to get the documentation for that one field I'm looking for. Especially because I don't even know if that documentation is still going to come. If I'm just looking at that one field, how am I supposed to know there's more?

This obviously applies to all the toml blocks.

Reply:
But the whole point was to make it like that so you can copy sections into the config and probably assemble the config out of man page pretty reliably?  I don't see any issue with sections like that.

Reply:
I **really** don't like that which is why I didn't write the manpage like this to begin with.

I think it's a bloody mess and makes the entire manpage unreadable.